cluster/plan.go

```diff
@@ -281,6 +283,12 @@ func (c *Cluster) BuildKubeAPIProcess(host *hosts.Host, serviceOptions v3.Kubern
 			fmt.Sprintf("%s=%s", AuditLogConfigSumEnv, getStringChecksum(string(bytes))))
 	}
 
+	matchedRange, _ := util.SemVerMatchRange(c.Version, util.SemVerK8sVersion122OrHigher)
```

Comment:
I didn't because the places that allow for returning the error will and it will be caught (at least that was the thought)

Reply:
I'm a bit confused still. Is the idea that even if an error occurs, the consumers of `matchedRange` will know that the returned value is a "failed" one? If that's the case, maybe we shouldn't even return an error at all?

Reply:
The function does not allow to properly return errors so I removed it and depended on the other invocations (as we added this in many places), but I added log lines for it because it is probably more clear with a log than just swallowing errors.